cpp/open3d/t/geometry/kernel/Metrics.h

```diff
@@ -0,0 +1,22 @@
+// ----------------------------------------------------------------------------
+// -                        Open3D: www.open3d.org                            -
+// ----------------------------------------------------------------------------
+// Copyright (c) 2018-2024 www.open3d.org
+// SPDX-License-Identifier: MIT
+// ----------------------------------------------------------------------------
+
+#include <vector>
+
+#include "open3d/core/Tensor.h"
+#include "open3d/t/geometry/Geometry.h"
+namespace open3d {
+namespace t {
+namespace geometry {
+
+core::Tensor ComputeMetricsCommon(core::Tensor distance12,
```

Comment:
Need to define here that distancexx is the Euclidean distance

Reply:
Comment added.